Source/WebCore/platform/graphics/displaylists/DisplayListRecorder.cpp

```diff
@@ -545,6 +545,7 @@ void Recorder::drawControlPart(ControlPart& part, const FloatRoundedRect& border
 
 void Recorder::resetClip()
 {
+    currentState().clipBounds = FloatRect::infiniteRect();
```

Comment:
I wonder if this is testable in any way?

Reply:
I think I can add a test to `DisplayListRecorderTests.cpp`. I will look at that as the review is on-going

Reply:
I added a test over in https://github.com/WebKit/WebKit/pull/13734

I will remove these changes as the two changes are not really dependent on each other and there were already separate bugs tracking them